### ISSUE TITLE
Static content is not served in Behave tests

### ISSUE THREAD

**Issue by rasmuse** [demokratikollen/demokratikollen#64]
The behave testing begins with starting a gunicorn server,

```
gunicorn --debug -b 127.0.0.1:5555 demokratikollen.www.gunicorn_testing:app
```

Then, all the tests (on `develop branch` we find it all in `static.py`) use something like

```
context.driver.get("http://127.0.0.1:5555/")
```

So the test driver receives the html correctly. Then the test driver requests the static content e.g., `http://127.0.0.1:5555/js/jquery.min.js`, which leads to the gunicorn server, which results in a 404 from the Flask app. The Flask app doesn't have routes to the static content, and it shouldn't either.

**The easiest solution** in my eyes is to direct all the behave tests to the nginx server on port 5000, which will then serve static content and redirect any dynamic stuff to gunicorn. (I just tested that setup on my dev machine.) Is there anything stopping us from this? I can come up with three potential problems:
1. I realize that the development gunicorn server would hove to be (manually) started before running `behave`, but this is not really a problem for me.
2. We still want the tests to run on Wercker too, and I know too little about Wercker/deploy/etc to see if the proposed solution.
3. Maybe the nginx server is set up to do some caching that we don't want in the develop-test-cycle?

@joharohl, I'm assigning you to this, as you are the Undisputed Master of Toolchains.

**Comment by joharohl** [comment]
I had a feeling this would happen :)

I think I know what to do.

2014-11-17 12:04 GMT+00:00 rasmuse notifications@github.com:

**Comment by joharohl** [comment]
Also fixed since 84753ce9bef38f40471f87e58407c81f753ff15e
